ci/android-install-sdk.sh

```diff
@@ -49,8 +49,8 @@ esac;
 
 # Try to fix warning about missing file.
 # See https://askubuntu.com/a/1078784
-mkdir -p .android
-touch .android/repositories.cfg
+mkdir ./.android/
+touch ./.android/repositories.cfg
```

Comment:
`touch` should re redundant because of `echo` below and the path should be probably `/root/.android`.

Reply:
Tried that (https://github.com/rust-lang/libc/pull/1344/commits/6b432e4a7d2a3f05414aa0a51bb3c53a9cf626f0) did not work.

Reply:
Are you sure that directory was present?